.github/workflows/node.js.yml

```diff
@@ -0,0 +1,30 @@
+# This workflow will do a clean installation of node dependencies, cache/restore them, build the source code and run tests across different versions of node
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-nodejs
+
+name: Node.js CI
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [21.x, 20.x, 19.x]
```

Comment:
This is a nice way to test your code on multiple Node versions, which is very good if you're writing code that needs to work on multiple versions, e.g. a library.

For an application like this, there is no real need to test on multiple node versions. It just means the tests take three times as long to run.